bin/sdk/gutenberg.js

```diff
@@ -20,6 +20,31 @@ const omitPlugins = [
 	webpack.HotModuleReplacementPlugin,
 ];
 
+const wordpressPackages = [
+	'api-fetch',
+	'block-serialization-spec-parser',
+	'blocks',
+	'components',
+	'data',
+	'editor',
+	'element',
+	'i18n',
+];
+
+const getWordPressExternals = () =>
+	wordpressPackages.reduce( ( externals, package ) => {
+		externals[ `@wordpress/${ package }` ] = {
+			window: [
+				'wp',
+				package.replace(
+					/-([a-z])/g,
+					( match, letter ) => letter.toUpperCase()
+				)
```

Comment:
Can we use lodash's [`camelCase`](https://lodash.com/docs/4.17.10#camelCase) here?

Reply:
Oh, I just found where this is probably coming from https://github.com/WordPress/gutenberg/blob/2c0510e4c396ba000e3744cd6ac58a9f6b1638ea/webpack.config.js#L61-L76

Reply:
Okay, so I take it that lodash's `camelCase` would break the `i18n` package name(?)

Reply:
Good sleuthing 👍 

Yeah, it would become `i18N` instead of `i18n`. Actually I did try `_.camelCase` before resorting to this solution.

Reply:
Maybe add a comment to the code, much like Gutenberg does?

Reply:
👍 https://github.com/Automattic/wp-calypso/pull/26549/commits/b183d270775a761af9c6d65f96bd0c1c824dd321